tests/softlearning/environments/adapters/softlearning_env_test.py

```diff
@@ -11,32 +11,49 @@ def test_observation_space(self):
         env = self.create_adapter()
         observation_space = env.observation_space
         self.assertTrue(
-            isinstance(observation_space, spaces.box.Box))
+            isinstance(observation_space, (spaces.Box, spaces.Dict)))
+        #TODO: Test actual conversion of dimensions and types of inside items; not just outside type
```

Comment:
This might be something you want to implement. If not, change the comment to be `# TODO(Alacarter): Test actual conversion of dimensions and types of inside items; not just outside type`.